documentation/data/data-msg-example.md

```diff
@@ -125,34 +125,34 @@ In this example, the Data message contains a value for the DeviceStatus property
 	}
 
 Numeric values must be used for properties defined as `enum` type using `reftypeid`.
-	
+
 ### Type-less Static Data Message
 
 In this example, the Data message extends the 'Tank' type to include a 'Maintenance Schedule' for this instance. The 'Tank' Type itself is not extended.
 
-	[{ 
+	[{
 		"typeid": "Tank",
-		"properties": { 
+		"properties": {
 			"Maintenance Schedule": {
 				"type": "array",
```

Comment:
Not related to this PR but it might be worth changing this example to include "Hourly Maintenance Schedule" for instance instead of currently used "Maintenance schedule" which is of type array that is unsupported in PWA at this point.

Reply:
I have replaced the array property. However, I noticed that this was the only place where we demonstrated how a property of type `array` works (with the additional `items` keyword), so I also added a new TYPE example.

Reply:
Sounds good. Thank you, Max.